packages/cli/src/commands/capabilities/create.ts

```diff
@@ -123,6 +129,10 @@ export default class CapabilitiesCreate extends InputOutputAPICommand<Capability
 				type: 'input',
 				name: 'basicCommandName',
 				message,
+				validate: (input) => {
+					// empty string is allowed here because it ends basic command name input
+					return !input || commandOrAttributeNameValidator(input)
```

Comment:
Does the input check and the corresponding comment make more sense in the validator function?

Reply:
This is the only place were input is optional; all other uses of `commandOrAttributeNameValidator` require input.